### PR TITLE
Move to serverless framework 3.

### PR DESCRIPTION
**Pull request by htorianik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-log-forwarding",
-  "version": "1.4.0",
+  "version": "1.5.0",
```
Comment (rddimon) [inline package.json:3]: Let's set the version `2.0.0` as we are changing the major version of serverless

File: test/integration-tests/test-utilities.js [hunk 2]
```diff
@@ -19,9 +19,8 @@ async function exec(cmd) {
   console.debug(`\tRunning command: ${cmd}`);
   return new Promise((resolve, reject) => {
     shell.exec(cmd, { silent: false }, (err, stdout, stderr) => {
-      const error = err || stderr;
-      if (error) {
-        return reject(error);
+      if (err) {
+        return reject(stderr);
```
Comment (rddimon) [inline test/integration-tests/test-utilities.js:23]: Is this correct ? if `error` reject `stderr`?

File: test/integration-tests/test-utilities.js [hunk 3]
```diff
@@ -92,8 +91,6 @@ async function destroyResources(url) {
     console.debug('\tResources Cleaned Up');
   } catch (e) {
     console.debug('\tFailed to Clean Up Resources');
-    console.debug(e);
-    throw e;
```
Comment (rddimon) [inline test/integration-tests/test-utilities.js:96]: Do we need to remove `throw e;`?

File: test/integration-tests/test-utilities.js (hunk 2 above)
Comment (htorianik) [reply test/integration-tests/test-utilities.js:23]: Yes, it is. If we return code of a bash command is not 0 then we reject a promise with error message (usually stderr)

File: package.json (hunk 1 above)
Comment (htorianik) [reply package.json:3]: Alex want to change a major version after moving to TS

File: test/integration-tests/test-utilities.js (hunk 2 above)
Comment (rddimon) [reply test/integration-tests/test-utilities.js:23]: So maybe rename `err` to `errorCode`

**Review by aoskotsky-amplify — APPROVED** [review]

**Pull request merged**